src/ape_init/_cli.py

```diff
@@ -33,6 +33,43 @@ def cli(cli_ctx, github):
             else:
                 folder.mkdir(exist_ok=False)
 
+        body = """
+# User Specified Ignored
+
+
+# Ape stuff
+.build/
+.pytest_cache
+__pycache__
+.env
+.cache
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# pyenv
+.python-version
+
+# Distribution / packaging
+.Python
+build/
+eggs/
+.eggs/
+*.egg-info/
+*.egg
+"""
+
+        git_ignore_path = project_folder / ".gitignore"
+        with git_ignore_path.open("w", encoding="utf-8") as f:
```

Comment:
This might overwrite what a user already has

Reply:
I added a conditional like the others